### ISSUE TITLE
readers.ept doesn't recognize polygon SRS

### ISSUE THREAD

**Issue by SmithB** [PDAL/PDAL#3276]
When provided a polygon WKT in the same SRS as the ept point cloud, pdal pipeline complains about a null SRS.

```
$ pdal pipeline grand_mesa_poly_pipeline.json -v 7
(PDAL Debug) Debugging...
(pdal pipeline readers.ept Debug) PDAL: readers.ept: Geometry::transform() failed.  NULL source SRS.
```
The pipeline file was:
```
{"pipeline": [{"polygon": "POLYGON ((-11978466.24356776 4741272.661908591, -11978462.29005747 4741311.479161278, -11978443.78277075 4741345.828683659, -11978413.53927421 4741370.48107233, -11978376.16386606 4741381.68322459, -11978337.34661337 4741377.729714306, -11978315.35146683 4741365.878880424, -11978295.71637716 4741381.891158504, -11978258.34341088 4741393.101454748, -11978219.52529763 4741389.156402991, -11978185.17174326 4741370.656601601, -11978160.51276499 4741340.418477639, -11978149.30246875 4741303.045511368, -11977839.41097429 4738053.193763002, -11977843.35602604 4738014.37564975, -11977861.85582743 4737980.022095374, -11977892.0939514 4737955.36311711, -11977929.46691767 4737944.152820867, -11977968.28503092 4737948.097872623, -11977990.72192173 4737960.180407645, -11978010.66883178 4737943.921078021, -11978048.04423993 4737932.71892576, -11978086.86149262 4737936.672436045, -11978121.211015 4737955.179722772, -11978145.86340367 4737985.423219305, -11978157.06555593 4738022.798627458, -11978466.24356776 4741272.661908591))", "filename": "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CO_MesaCo_QL2_2015_LAS_2017/ept.json", "type": "readers.ept", "tag": "readdata"}, {"limits": "Classification[2:2]", "type": "filters.range", "tag": "classify"}, "USGS_LPC_CO_MesaCo_QL2_2015_LAS_2017_ATL06_20190412021205_02110306_003_01.laz"]}
```
A similar query using a bounds string in the same SRS produced good results:
```
bounds:"([-12062642, -11996650], [4706074, 4759865])"
```
</details>

**Expected behavior**
I would have expected a polygon in the same SRS as the ept file to produce a clipped output

**System/installation information:**

```
$ pdal --version
-------------------------------------------------------------------
pdal 2.2.0 (git-version: 47145b)
--------------------------------------------------------------------
```

**Comment by abellgithub** [comment]
Confirmed.

**Comment by abellgithub** [comment]
Fixed by #3277
